cmd/juju/commands/main.go

```diff
@@ -70,6 +70,44 @@ or your local machine.
 See https://discourse.jujucharms.com/ for ideas, documentation and FAQ.
 `
 
+var usageHelp = `
+Usage: juju [help] <command>
+
+Summary:
+Juju is model & application management software designed to leverage the power
+of existing resource pools, particularly cloud-based ones. It has built-in
+support for cloud providers such as Amazon EC2, Google GCE, Microsoft
+Azure, OpenStack, and Rackspace. It also works very well with MAAS and
+LXD. Juju allows for easy installation and management of workloads on a
+chosen resource pool.
+
+See https://jujucharms.com/docs/stable/help for documentation.
+
+Common commands:
```

Comment:
We probably want to review these at some point, perhaps split them into more sections (based on functionality?) and include some of the charmhub-related commands which will be available with the 2.9 release.